SpeechResponder/Service/ScriptRecoveryService.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
```

Comment:
`using System.Collections.Generic;` and `using System.Text;` both appear to be unused. Remove unused usings?

Reply:
If you are OK with it, I would refactor the whole EditScriptWindow.cs accordingly to MS conventions.

Reply:
I'm not sure what that would entail. In any case, let's consider that on a separate PR once the current PR is merged.